api/v1beta1/mysqlcluster_webhook.go

```diff
@@ -22,8 +22,6 @@ func (r *MySQLCluster) SetupWebhookWithManager(mgr ctrl.Manager) error {
 
 // EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
 
-//+kubebuilder:webhook:path=/mutate-moco-cybozu-com-v1beta1-mysqlcluster,mutating=true,failurePolicy=fail,sideEffects=None,groups=moco.cybozu.com,resources=mysqlclusters,verbs=create,versions=v1beta1,name=mmysqlcluster.kb.io,admissionReviewVersions=v1
```

Comment:
Why don't we remove this file altogether?

Reply:
Thanks！, I remove it: https://github.com/cybozu-go/moco/pull/350/commits/c7988e3dda9a4053a9a79f0ccd19cc62b5a762e0

But, a conversion webhook is necessary, so we left the webhook registration process.